BookingQueueSubscriber/BookingQueueSubscriber.Services/VideoWeb/VideoWebService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using VideoApi.Contract.Requests;
+using Microsoft.Extensions.Logging;
+using System.Net.Http;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using System.Net.Http.Headers;
+using System.IdentityModel.Tokens.Jwt;
+using Microsoft.IdentityModel.Tokens;
+using System.Security.Claims;
+
+namespace BookingQueueSubscriber.Services.VideoWeb
+{
+    public class VideoWebService : IVideoWebService
+    {
+        private readonly HttpClient httpClient;
+        private readonly ILogger<VideoWebService> logger;
+        public VideoWebService(HttpClient _httpClient, ILogger<VideoWebService> _logger)
+        {
+            httpClient = _httpClient;
+            logger = _logger;
+        }
+        public async Task PushParticipantsAddedMessage(Guid conferenceId, AddParticipantsToConferenceRequest request)
+        {
+            var path = $"internalevent/ParticipantsAdded?conferenceId={conferenceId}";
```

Comment:
This will need to be changed to use a client similar to VideoApiClient in the future. Discussed with @kevin-hmcts